tests/routes/test_admin.py

```diff
@@ -24,262 +22,22 @@
     Instrument,
     Category,
     Keyword,
-    User,
-    PasswordResetToken,
 )
 from app.db.models.law_policy import Geography
 
 
-def test_get_users(client, test_superuser, superuser_token_headers):
```

Comment:
Do we need to replace these tests with some basic tests to use the new models post db update?

Reply:
All this functionality has been removed - user management will be manual for a while, so there's no code to test. We will probably need these sorts of tests again, but that's a task for future-us